packages/contracts-bedrock/snapshots/storageLayout/L1CrossDomainMessenger.json

```diff
@@ -7,18 +7,11 @@
     "type": "address"
   },
   {
-    "bytes": "1",
-    "label": "_initialized",
+    "bytes": "12",
+    "label": "spacer_0_20_12",
     "offset": 20,
     "slot": "0",
-    "type": "uint8"
-  },
-  {
-    "bytes": "1",
-    "label": "_initializing",
-    "offset": 21,
-    "slot": "0",
-    "type": "bool"
+    "type": "bytes12"
```

Comment:
these two spacers both occupy a single slot. They could be consolidated now or in the future.